resources/js/ClickAndHold.js

```diff
@@ -0,0 +1,46 @@
+//code borrowed from: https://www.youtube.com/watch?v=A95mIE2HdcY
```

Comment:
Is this class still needed, if `Board.js` implements the `_onHoldStart` and `onHoldEnd` functions? I can see `app.js` importing this class, but I can't see it being used anywhere

Reply:
No its no longer needed. I can remove it and add the citation to those specific functions.